src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -154,6 +165,10 @@ public function startTest(\PHPUnit_Framework_Test $test)
         }
     }
 
+    /**
+     * @param \PHPUnit_Framework_Test $test
```

Comment:
We dont add doc blocks that only repeat what the signature already says, the all should be removed.